src/components/FormView.jsx

```diff
@@ -1,5 +1,6 @@
 import React, { useCallback, useState, useEffect } from 'react';
 import { useIntl, defineMessages } from 'react-intl';
+import HCaptcha from '@hcaptcha/react-hcaptcha';
 import { GoogleReCaptcha } from 'react-google-recaptcha-v3';
```

Comment:
How about a dynamic import based on the provided env vars?

Reply:
I think wrapping in loadable() would be just fine.

Reply:
Yeah that's right actually